SU-HSDO/suhumsci
# feat(STN-155):  3 column layout and styling refinement

# READY FOR REVIEW

## Summary
**[JIRA CARD 155](https://sparkbox.atlassian.net/browse/STN-155)**

Adds template and refined styles for 3 column layout, and 3 column with image styling tweaks.

See screenshots in a comment below for visual design with background colors for extra clarity.

## Steps to Test
1. Use layout builder to add 3 or 4 layouts using 3 column layout. Add one with only main content, one with 1 sidebar, and 1 with 2 sidebars.
2. Inside `humsci_basic` run `npm run test` and make sure all tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
